kubernetes/chart/zulip/README.md.gotmpl

```diff
@@ -0,0 +1,58 @@
+# Zulip
+
+{{ template "chart.versionBadge" . }}{{ template "chart.typeBadge" . }}{{ template "chart.appVersionBadge" . }}
+
+[Zulip](https://zulipchat.com/), the world's most productive chat
+
+Helm chart based on https://github.com/zulip/docker-zulip
+
+## Installation
+
+Copy `values-local.yaml.example`, modify it as instructed in the comments, then
+install with the following commands:
+
+```
+helm dependency update                      # Get helm dependency charts
+helm install -f ./values-local.yaml zulip . # Install Zulip
+```
+
+This will show a message on how to reach your Zulip installation and how to
+create your first realm. Wait for all your pods to be ready before you continue.
+You can run `kubectl get pods` to their current state. Once all pods are ready,
+you can run the commands to create a Realm, and you can reach Zulip following
+the instructions as well.
+
+### Installing on Minikube
+
+To install this chart on [minikube](https://minikube.sigs.k8s.io/docs/):
+
+Copy `values-minikube.yaml.example` to `values-local.yaml` and add
+(auto-generated) passwords to it as instructed in the comments. Run `minikube
+start` to start minikube. Then follow the above installation instructions.
```

Comment:
The difficulty, at least with my play with minikube, is that because there's no SSL in the system, and the CSRF cookie is set as `Secure`, you can't get past the first form submission of the realm creation screen.

Did you successfully get this working in minikube?

Reply:
I don't think I tested more than just getting the access screen and getting all pods ready. I'm using the chart in production, though, with prod. certificates.

If there's a way I can let Zulip ignore self signed certificates or http:// traffic I can apply that to make it work in Minikube, but I don't know a lot about Zulip internals.

Reply:
OK, I added instructions on how to set up cert-manager inside minikube and use the ingress addon to make zulip available on `zulip.local`, with a self-signed certificate. It worked perfectly in my tests, so I hope you'll get success too. If not, please reach out!

AFAIK that should absolve the last remaining blocker to getting this merged :) I'm pretty proud of the chart so far, and I'm really happy with the improvements based on your feedback. Thanks a lot :)